chcemvediet/apps/anonymization/finalization.py

```diff
@@ -75,10 +75,10 @@ def finalize_using_libreoffice(attachment_anonymization):
 
 def finalize_attachment():
     attachment_anonymization = (AttachmentAnonymization.objects
-                                .filter(successful=True,
-                                        content_type=content_types.ODT_CONTENT_TYPE,
-                                        attachment__attachmentfinalization__isnull=True)
-                                .first())
+            .successful()
+            .anonnymized_to_odt()
```

Comment:
Si si to testoval?

```
[2019-08-30 13:27:25,135] poleno.cron ERROR Cron job "anonymization.anonymization" failed:
projekty/00.moje/2013-09-23.chcemvediet/chcemvediet/env/local/lib/python2.7/site-packages/django_cron/management/commands/runcrons.py", line 57, in run_cron_with_cache_check
    manager.run(force)
  File "/home/martin/projekty/00.moje/2013-09-23.chcemvediet/chcemvediet/env/local/lib/python2.7/site-packages/django_cron/__init__.py", line 175, in run
    self.msg = self.cron_job.do()
  File "/home/martin/projekty/00.moje/2013-09-23.chcemvediet/chcemvediet/poleno/cron/__init__.py", line 37, in do
    return function()
  File "/home/martin/projekty/00.moje/2013-09-23.chcemvediet/chcemvediet/chcemvediet/apps/anonymization/cron.py", line 14, in anonymization
    finalize_attachment()
  File "/home/martin/projekty/00.moje/2013-09-23.chcemvediet/chcemvediet/chcemvediet/apps/anonymization/finalization.py", line 77, in finalize_attachment
    attachment_anonymization = (AttachmentAnonymization.objects
AttributeError: 'AttachmentAnonymizationQuerySet' object has no attribute 'anonnymized_to_odt'
```

Reply:
Ano testoval.
Sorry, len typo chybicka pri prepisovani.